### ISSUE TITLE
Invalid imports in test files

### ISSUE THREAD

**Issue by ridenaio** [ljlm0402/typescript-express-starter#190]
### Summary (요약)

I created a new project with prisma template. All tests are passing, but imports are not visible. I use latest VS code. 

How to this these imports (problem exists only in .test.ts files).

![Untitled](https://user-images.githubusercontent.com/50695987/199468545-9195b9df-8f63-4699-a2ae-faad157bd7eb.png)

**Comment by Mounir-Bennacer** [comment]
you have to use ../app instead of @/ and you should be good to go

**Comment by ridenaio** [comment]
Yes, I can, but this is the template, it should work or maybe need to be fixed?

Moreover is /src folder these imports like @/app, @/utils works perfectly. What's the difference?

**Comment by tiavina-mika** [comment]
I have the same issue with the default template:
![image](https://user-images.githubusercontent.com/42656064/202834990-76e59320-5993-4616-85dd-2800c757ea64.png)

Is there another options to fix it than manually?

**Comment by tiavina-mika** [comment]
I fix it by removing `src/tests` in `exclude `in` tsconfig.json`

**Comment by dap23** [comment]
try 
import { App } from '@/app';

**Comment by dap23** [comment]
or just edit class App in app.ts
to export default class App

**Comment by ljlm0402** [comment]
We will close this issue with the release of a new version.
